lib/Remote/RemoteTargetLocator.php

```diff
@@ -84,7 +84,7 @@ public function frame($frame)
      */
     public function window($handle)
     {
-        $params = ['name' => (string) $handle];
+        $params = ['handler' => (string) $handle];
```

Comment:
However keep in mind the compatibility with the old protocol must be kept - so there should be somewhere condition so that this is executed differently in JsonWire OSS protocol W3C WebDriver protocol. Have look at method `activeElement() ` in the same file, where something similar is done.

Reply:
Thanks @OndraM ,

I've used the same process as in activeElement.

I wonder if there's any merit to having it take `$isW3cCompliant` into the constructor like in RemoteMouse and doing that consistently across the board.

Let me know if you want me to make that change, or whether it's worth raising a separate issue for it.